libsql/tests/integration_tests.rs

```diff
@@ -650,3 +653,102 @@ async fn deserialize_row() {
     assert_eq!(data.status, Status::Draft);
     assert_eq!(data.wrapper, Wrapper(Status::Published));
 }
+
+#[tokio::test]
+#[ignore]
+// fuzz test can be run explicitly with following command:
+// cargo test vector_fuzz_test -- --nocapture --include-ignored
+async fn vector_fuzz_test() {
+    let mut global_rng = rand::thread_rng();
+    for attempt in 0..10000 {
+        let seed = global_rng.next_u64();
+
+        let mut rng =
+            rand::rngs::StdRng::from_seed(unsafe { std::mem::transmute([seed, seed, seed, seed]) });
+        let db = Database::open(":memory:").unwrap();
+        let conn = db.connect().unwrap();
+        let dim = rng.gen_range(1..=1536);
+        let operations = rng.gen_range(1..128);
+        println!(
+            "============== ATTEMPT {} (seed {}u64, dim {}, operations {}) ================",
+            attempt, seed, dim, operations
+        );
+
+        let _ = conn
+            .execute(
+                &format!(
+                    "CREATE TABLE users (id INTEGER PRIMARY KEY, v FLOAT32({}) )",
+                    dim
+                ),
+                (),
+            )
+            .await;
+        // println!("CREATE TABLE users (id INTEGER PRIMARY KEY, v FLOAT32({}) );", dim);
```

Comment:
Why not make these printouts unconditional?

Reply:
They can be pretty large (I generate embeddings of dimensions up to 1536).

The idea is that you can run fuzz test and wait for exit. If failure happen you can uncomment these lines, copy seed to the source code and reproduce failure.